supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
```

Comment:
So "Links" is the main category here, but we still need to make individual entries below it that follow this template: https://github.com/redhat-documentation/supplementary-style-guide/blob/master/GUIDELINES.adoc#contributing-style-guideline-entry

Can we take a look at these pieces of guidance and determine whether any can be broken out by internal vs. external links? And then have an entry for "Internal links" and one for "External links"? For example, the list of benefits for internal linking seems to only apply for internal. Though those aren't really guidelines, so I'm not sure if they really belong here.

Or another option would be to break it out and have the technical best practices section be its own entry, and then have the anchor text (or link text as I suggest elsewhere) be its own entry.

wdyt?